osquery/remote/transports/tls.cpp

```diff
@@ -37,7 +37,7 @@ CLI_FLAG(string,
 /// Path to optional TLS server/CA certificate(s), used for pinning.
 CLI_FLAG(string,
          tls_server_certs,
-         "",
+         "/usr/share/osquery/certs/certs.pem",
```

Comment:
I'm also down to change this from `certs.pem` to `curl.pem` or something if anyone would prefer this to be something more meaningful than `certs.pem`.